src/pybind/mgr/dashboard/frontend/src/styles/defaults/_bootstrap-defaults.scss

```diff
@@ -1,8 +1,72 @@
 @use '../vendor/variables';
 
-// Bootstrap defaults
-
+// Color system
+
+$white: #fff !default;
+$gray-100: #f8f9fa !default;
+$gray-200: #e9ecef !default;
+$gray-300: #dee2e6 !default;
+$gray-400: #ced4da !default;
+$gray-500: #adb5bd !default;
+$gray-600: #6c757d !default;
+$gray-700: #495057 !default;
+$gray-800: #343a40 !default;
 $gray-900: #212529 !default;
+$black: #000 !default;
+
+$blue: #007bff !default;
+$indigo: #6610f2 !default;
+$purple: #6f42c1 !default;
+$pink: #a94442;
+$red: #ef5c55;
+$orange: #fd7e14 !default;
+$yellow: #ffc200;
+$green: #0b0;
+$teal: #20c997 !default;
+$cyan: #17a2b8 !default;
+
+$primary: #2b99a8;
+$secondary: #374249;
+$success: $green;
+$info: $primary;
+$warning: $yellow;
+$danger: $red;
+$light: $gray-100;
+$dark: #777;
```

Comment:
Before those changes `secondary` was the ceph red tone, which became the accent color. We didn't change the value of `dark`. Primary and secondary should be the main brand colors of ceph. We could set `dark` to the default value of bootstrap which would be `gray-800 (#343A40)`, but this will definitely change the optic of some elements.